packages/client3/components/design-system/input/input.tsx

```diff
@@ -147,15 +147,16 @@ export const Input = forwardRef<HTMLInputElement, InputProps>(function Input(
         ) : null}
       </div>
       {helperText || _errorMessage ? (
-        <HelperText
-          className={clsx(
-            isError ? "text-state-error" : "text-dark-80",
-            "mt-1 text-sm leading-none"
-          )}
-        >
-          {_errorMessage ? _errorMessage : helperText}
-        </HelperText>
-      ) : null}
+        <></>
+      ) : // <HelperText
+      //   className={clsx(
+      //     isError ? "text-state-error" : "text-dark-80",
+      //     "mt-1 text-sm leading-none"
+      //   )}
+      // >
+      //   {_errorMessage ? _errorMessage : helperText}
+      // </HelperText>
+      null}
```

Comment:
Old component used by goldfinch. I commented instead of deleting for future references.

Reply:
Up to you whether our reference will be client2 or have these comments in client3